Module1/todo_app/app.py

```diff
@@ -1,105 +1,132 @@
 from flask import Flask, render_template, request, redirect, make_response, url_for
+from todo_app.data.card_items import Item
+from todo_app.data.view_model import ViewModel
 
 from todo_app.flask_config import Config
 from todo_app.data.session_items import get_items, add_item, save_item
 from todo_app.data.trello_items import *
 
-app = Flask(__name__)
-app.config.from_object(Config())
-
 API_KEY = os.getenv('API_KEY')
 API_TOKEN = os.getenv('API_TOKEN')
 BOARD_NAME = os.getenv('BOARD_NAME')
 
-""" BOARD_LIST = {
-    'To_Do': '64ef5905b30edf49e85fa346', 
-    'Doing': '64ef5905b30edf49e85fa347',
-    'Done': '64ef5905b30edf49e85fa348'
-}   """
-
 trello_instance = MYTRELLO(API_KEY, API_TOKEN)
+BOARD_LIST = trello_instance.lists_on_board()
```

Comment:
When your app runs in test mode, I see these lines error - that's because they run when this file is first imported, which happens before the tests have had time to set up e.g. the mocking, and load in the fake environment variables. Moving these lines (from `API_KEY = ...` to `BOARD_LIST = ...`) into the start of your `create_app` function might be a sensible first step, so that we're explicitly stating they should only run when the app is actually being created, rather than when the file is imported